packages/dds/tree/src/edit-manager/editManager.ts

```diff
@@ -326,6 +344,25 @@ export class EditManager<
         }
         return this.branches.get(sessionId) as Branch<TChangeset>;
     }
+
+    private inverseFromCommit(commit: Commit<TChangeset>): TaggedChange<TChangeset> {
+        return {
+            revision: revisionTagForInverse(brand(commit.seqNumber)),
+            change: this.changeFamily.rebaser.invert(taggedChangeFromCommit(commit)),
+        };
+    }
+
+    private allocateLocalRevisionTag(): RevisionTag {
+        return brand(this.nextLocalRevision--);
+    }
+}
+
+function taggedChangeFromCommit<T>(commit: Commit<T>): TaggedChange<T> {
+    return { revision: brand(commit.seqNumber), change: commit.changeset };
+}
+
+function revisionTagForInverse(revision: RevisionTag | undefined): RevisionTag | undefined {
+    return undefined;
```

Comment:
It would be great to have some comments (either here or linked from here) explaining why it's OK for us to use `undefined` in all such cases.
(I haven't managed to convinved myself that it is, but it seems better to wait for the explanation first)

Reply:
I will add a comment, but this function is a placeholder in case we find we do need to provide identities for inverse changes. I don't currently see how these identities would be useful, so I'm leaving them undefined for now.